Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -147,6 +149,9 @@ public boolean onTouch(View v, MotionEvent event)
 				{
 					NativeLibrary.onGamePadEvent(NativeLibrary.TouchScreenDevice, button.getId(), ButtonState.RELEASED);
 				}
+
+				button.setPressedState(false);
+				invalidate();
```

Comment:
Shouldn't these go inside the if statement like they are in the down press events?